src/types/bloom_filter.h

```diff
@@ -84,6 +84,13 @@ class BlockSplitBloomFilter {
   /// @return false if the number of bytes of Bloom filter bitset is not a power of 2, and true means successfully init
   bool Init(std::string bitset);
 
+  /// Create the non-owned BlockSplitBloomFilter. It use the bitset as underlying bitset. It is the caller's
+  /// responsibility to ensure the bitset would not to change.
+  ///
+  /// @param bitset The given bitset for the Bloom filter underlying bitset.
+  /// @return the unique_ptr of the const non-owned BlockSplitBloomFilter
+  static std::unique_ptr<const BlockSplitBloomFilter> CreateNonOwned(const std::string& bitset);
```

Comment:
why is it called "non-owned"?

Reply:
Because it create a non-owned slice version BloomFilter. The underlying bitset of BloomFilter is a reference string from caller. Maybe we should call it "ReadOnly" or others?

Reply:
Oh, I think it doesn‘t work as expected, I will change it.